scss/_card.scss

```diff
@@ -241,6 +241,9 @@
     // Handle rounded corners
     @if $enable-rounded {
       &:first-child {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
```

Comment:
You can use the [`border-<side>-radius` mixins](https://github.com/twbs/bootstrap/blob/69f92408630707fe704eae309a71867adefa3d39/scss/mixins/_border-radius.scss) here.